src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,35 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.getSessionFactory().openSession();
```

Comment:
You are calling `factory.getSessionFactory().openSession()` which is incorrect because `factory` is already a `SessionFactory` instance. You should call `factory.openSession()` instead.